Formula/d/devd.rb

```diff
@@ -1,76 +0,0 @@
-class Devd < Formula
-  desc "Local webserver for developers"
-  homepage "https://github.com/cortesi/devd"
-  license "MIT"
-  head "https://github.com/cortesi/devd.git", branch: "master"
-
-  stable do
-    url "https://github.com/cortesi/devd/archive/refs/tags/v0.9.tar.gz"
-    sha256 "5aee062c49ffba1e596713c0c32d88340360744f57619f95809d01c59bff071f"
-
-    # Get go.mod and go.sum from commit after v0.9 release.
-    # Ref: https://github.com/cortesi/devd/commit/4ab3fc9061542fd35b5544627354e5755fa74c1c
-    # TODO: Remove in the next release.
-    resource "go.mod" do
-      url "https://raw.githubusercontent.com/cortesi/devd/4ab3fc9061542fd35b5544627354e5755fa74c1c/go.mod"
-      sha256 "483b4294205cf2dea2d68b8f99aefcf95aadac229abc2a299f4d1303f645e6b0"
-    end
-    resource "go.sum" do
-      url "https://raw.githubusercontent.com/cortesi/devd/4ab3fc9061542fd35b5544627354e5755fa74c1c/go.sum"
-      sha256 "3fb5d8aa8edfefd635db6de1fda8ca079328b6af62fea704993e06868cfb3199"
-    end
-  end
-
-  bottle do
-    rebuild 4
-    sha256 cellar: :any_skip_relocation, arm64_sonoma:   "f3fc464a6deb3e098bf848f03c4cd5030287493c92198140ad9056e36b67a09e"
-    sha256 cellar: :any_skip_relocation, arm64_ventura:  "13847da393a2fc68810f987451c78f751fd9a3ea4fc2e07a00b01464f3eed02c"
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "82d937b6bcf0f37755df12694e934a8032ce155fc2895ec227f4887b2662d9c2"
-    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "47f06f9c6157c81eaf52040448e5f131a6a660f96e78186146ab469345e6ea96"
-    sha256 cellar: :any_skip_relocation, sonoma:         "144a24494c3e24414d13e0d336b09d69969b110cfaf37c618d4e2d7c599a0054"
-    sha256 cellar: :any_skip_relocation, ventura:        "c3a45de493d66241a712df0ec91b66bbb110a221e9e5b4d0216050dea7cc7e9e"
-    sha256 cellar: :any_skip_relocation, monterey:       "e815c896205297337741c856016809aa6603547c4a4302acb0ad307f3c91f10c"
-    sha256 cellar: :any_skip_relocation, big_sur:        "446557dc47076e2e0f4d93d6e33ecdac80721f4da9c391af29154509c425dd57"
-    sha256 cellar: :any_skip_relocation, catalina:       "0bfc6ccb8402282c4c3b3bf375cc3be882deba31f35a627801f3009456bf62f2"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "9c82a9adb431d63c5ac65d04c6f27592e6fe23a3ffa0e5a118fd34a113d32af4"
-  end
-
-  # Current release is from 2019-01-20 and needs deprecated `dep` to build.
-  # We backported upstream support for Go modules, but have not received
-  # a response on request for a new release since 2021-01-21.
-  # Issue ref: https://github.com/cortesi/devd/issues/115
-  disable! date: "2023-10-03", because: :unmaintained
```

Comment:
This is why it had zero installs in the last 90 days. Should we wait until 2024-10-03 before removing this?

https://docs.brew.sh/Deprecating-Disabling-and-Removing-Formulae#removal

Reply:
we talked about in the AGM that we dont have to wait for the full year.

Reply:
We don't, but there isn't really any reason not to, since they're automatically removed already.

I suggest updating the automated removal timeline if you'd rather disabled formulae were deleted sooner.

Reply:
yeah, I can also make some change for the deletion automation

Reply:
I'm trying to improve the deprecation docs: https://github.com/Homebrew/brew/pull/16829

> they can be manually removed three months after their disable date.

We should change this to: "automatically disabled", so we don't have to run manual actions. Let the bot do the hard work for us.

Reply:
> We should change this to: "automatically disabled"

You mean "automatically removed", right? Which also means the one-year automatic removal mentioned in the ["Removal"](https://docs.brew.sh/Deprecating-Disabling-and-Removing-Formulae#disabling) section would take place sooner.